packages/web/DEPRECATIONS-v2.md

```diff
@@ -0,0 +1,120 @@
+# Deprecations List
+
+Following deprecations and feature flags will be removed in version 2 of the _spirit-web_ package.
+
+> Please follow these steps to safely upgrade your design system to Spirit Design System v2 components.
+
+[What are deprecations?][readme-deprecations]
+
+[What are feature flags?][readme-feature-flags]
+
+## Feature Flags
+
+### Dropdown Shadow
+
+The feature flag enabling the dropdown shadow will be removed and the enhanced
+shadow will be enabled by default.
+
+## Modal Uniform Variant
```

Comment:
This heading is on the same level as the `Feature Flags`. I think this should be on a lower level. Same as another one.